custom_visibility_7_9/src/custom/data/visibility/FilterOpportunities.php

```diff
@@ -65,20 +65,30 @@ public function __construct(SugarBean $bean, $params = null)
     }
 
     /**
-     * {@inheritdoc}
+     * orvewrite parent method, SugarVisibility::addVisibilityWhere
```

Comment:
Same for 7.9.  Can you apply suggestions from custom_visibility to this section of the code as well?